torchx/schedulers/ray_scheduler.py

```diff
@@ -97,6 +100,15 @@ class RayJob:
 class RayScheduler(Scheduler):
     def __init__(self, session_name: str) -> None:
         super().__init__("ray", session_name)
+        # TODO: Users can run ray.status() to figure out the right URL 
+        # But for now Ray team will explore finding a programmatic way of offering this
+        self.client = JobSubmissionClient("http://127.0.0.1:8265")
```

Comment:
Not sure I am following this method body. I assume this will evolve into something that will be part of the `schedule()` method for resolving the URL of the Job REST API, right?